.github/ISSUE_TEMPLATE/documentation_report.yml

```diff
@@ -64,7 +64,7 @@ body:
           <em>Why?</em>
         </summary>
 
-        We would do it by ourselves but unfortunatelly, the curent
+        We would do it by ourselves but unfortunately, the current
```

Comment:
Is this statement necessary? When I try creating a docs issue, the only available option appears already selected.

Reply:
Yes, it hasn't been necessary in any of the forms for some time. I've been meaning to clean it up… Just drop it entirely.